jobdsl-gradle/src/main/groovy/job/JobBuilder.groovy

```diff
@@ -284,6 +301,31 @@ public class JobBuilder {
         }
         this
     }
+    /**
+     * Use this method to parameterize your build using NodeLabel parameter plung,
+     * see more details https://jenkinsci.github.io/job-dsl-plugin/#path/job-parameters-nodeParam
+     * and https://wiki.jenkins-ci.org/display/JENKINS/NodeLabel+Parameter+Plugin
+     * There is a reference to issue [JENKINS-33756](https://issues.jenkins-ci.org/browse/JENKINS-33756)
+     * "Run on all nodes matching the label" or allNodes in DSL flag schedules one more build
+     * on the node job has been started.
+     *
+     * @String paramName                  the paramater name
+     * @ArrayList<String> nodesLabels     list of labels of nodes you want to run the job
+     *
+     */
+    public JobBuilder addJobBuildParameter(String paramName, ArrayList<String> nodesLabels, boolean debug = false) {
```

Comment:
This is a too generic name - you are adding a very specific type of parameter. BTW, do we really need to wrap such one-liners? Would it be possible to just update the object when it is created